Doc/library/os.rst

```diff
@@ -3195,6 +3195,27 @@ features:
    .. versionadded:: 3.8
 
 
+.. function:: eventfd(initval[, flags=os.EFD_CLOEXEC])
```

Comment:
What's the rationale for `flags` defaulting to `EFD_CLOEXEC` instead of `0`?
Note: I'm not familiar with this syscall, but if flags=0 is a legitimate use case then I think it makes sense to have:
```
eventfd(initval[, flags=0])
```

Reply:
See https://www.python.org/dev/peps/pep-0446/

CPython tries hard to make all file descriptors as ``CLOEXEC`` by default. Users can opt-out from the default with a different flag.

Reply:
Makes sense.